Cargo.toml

```diff
@@ -52,7 +52,7 @@ path = "examples/deferred/main.rs"
 name = "flowmap"
 path = "examples/flowmap/main.rs"
 
-[[target.x86_64-pc-linux-gnu.example]]
+[[example]]
```

Comment:
Could we go the opposite way and make it build on windows? I'm using linux, so I can't really test anything, though.